<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFeaturePreservingMeshSimplification</class>
 <widget class="QDialog" name="DialogFeaturePreservingMeshSimplification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog Feature Preserving Mesh Simplification</string>
  </property>
  <widget class="QGroupBox" name="groupBox_NormalFilter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Normal Fiter</string>
   </property>
   <widget class="QDoubleSpinBox" name="BoxBFtheta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="BoxBFIteration">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Vertex Threshold：</string>
    </property>
    <property name="buddy">
     <cstring>BoxBFVertexThreshold</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="BoxBFVertexThreshold">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000100000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRunBilateralFilter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Bilateral Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'SimSun'PingFang SC,Lantinghei SC,Microsoft YaHei,arial,宋体,sans-serif,tahoma'; font-size:16px; color:#333333;&quot;&gt;θ:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteration：</string>
    </property>
    <property name="buddy">
     <cstring>BoxBFIteration</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Planar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Region Extraction</string>
   </property>
   <widget class="QPushButton" name="ButtonRunPlanarRegionExtraction">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Planar Region Extraction</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleFittingThreshold">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleAngle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Area Threshold:</string>
    </property>
    <property name="buddy">
     <cstring>doubleAreaThreshold</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fitting Threshold:</string>
    </property>
    <property name="buddy">
     <cstring>doubleFittingThreshold</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle Theta:</string>
    </property>
    <property name="buddy">
     <cstring>doubleAngle</cstring>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleAreaThreshold">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_QEM">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>QEM</string>
   </property>
   <widget class="QPushButton" name="ButtonRunQEM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run QEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Points Number:</string>
    </property>
    <property name="buddy">
     <cstring>BoxPointsNumber</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="BoxPointsNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="BoxEnergyValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy Value:</string>
    </property>
    <property name="buddy">
     <cstring>BoxEnergyValue</cstring>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
